### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by victor-yunenko**
Related to #976 

todo:
- [x] restore a missing profile image
- [x] test in edge

### CODE REVIEW

**Comment by victor-yunenko** [comment]
@sebgrebe. I finished everything, including the footer.

To me it still looks immature, but compared to our existing homepage I'm going to call this one deployable - the sooner the better. We can change the text or images afterwards ticket by ticket.

https://eahub-dev-815fc48.us.aldryn.io/

File: eahub/templates/components/navbar.html [hunk 1]
```diff
@@ -1,100 +1,141 @@
...
+                        href="https://resources.eahub.org/"
+                    >
+                        <i class="fa fa-book"></i> EA Resources
+                    </a>
+                </li>
+                <li class="nav-item d-flex d-lg-none">
+                    <a class="nav-link"
+                        href="https://donationswap.eahub.org"
+                    >
+                        <i class="fa fa-exchange-alt"></i> EA Donation Swap
```
Comment (sebgrebe) [inline eahub/templates/components/navbar.html:70]: Can we have external link symbols here?

**Review by sebgrebe — APPROVED** [review]
Code looks good, @vagarwalla could you do UX review please?

File: eahub/templates/components/navbar.html (hunk 1 above)
Comment (victor-yunenko) [reply eahub/templates/components/navbar.html:71]: done

I haven't invested a lot of time sadly, hence it looks a bit ugly (two icons on one line is somehow harder to make professional)

image

**Comment by vagarwalla** [comment]
I left a comment on #976

**Review by sebgrebe — DISMISSED** [review]
Approving again (I assume we should wait though on @vagarwalla giving thumbs up that her comments have been addressed before merging in)

**Pull request merged**